test/README.md

```diff
@@ -0,0 +1,129 @@
+Test OpenSSL
+============
+
+After a successful build, and before installing, the libraries should be tested.
+Run:
+
+    make test                                      # Unix
+    mms test                                       ! OpenVMS
+    nmake test                                     # Windows
+
+**Warning:** you MUST run the tests from an unprivileged account
+(or disable your privileges temporarily if your platform allows it).
+
+If some tests fail, take a look at the section Test Failures below.
+
+Test Failures
+-------------
+
+If some tests fail, look at the output.  There may be reasons for the failure
+that isn't a problem in OpenSSL itself (like an OS malfunction or a Perl issue).
+You may want increased verbosity, that can be accomplished like this:
+
+Full verbosity (`make` macro `VERBOSE` or `V`):
+
+    make V=1 test                                  # Unix
+    mms /macro=(V=1) test                          ! OpenVMS
+    nmake V=1 test                                 # Windows
+
+Verbosity on test failure (`VERBOSE_FAILURE` or `VF`, Unix example shown):
+
+    make test VF=1
+
+Verbosity on failed (sub-)tests only (`VERBOSE_FAILURES_ONLY` or `VFO`):
+
+    make test VFO=1
+
+Verbosity on failed (sub-)tests, in addition progress on succeeded (sub-)tests
+(`VERBOSE_FAILURES_PROGRESS` or `VFP`):
+
+    make test VFP=1
+
+If you want to run just one or a few specific tests, you can use
```

Comment:
Doesn't the rest of this paragraph belong to Running selected tests?

Reply:
Not really - at this point we have just one example of using `TESTS`, mostly to make the point in the following example that it may be combine with the verbosity specifiers which are the topic of this section.